scene/theme/theme_db.cpp

```diff
@@ -225,5 +348,54 @@ ThemeDB::~ThemeDB() {
 	fallback_icon.unref();
 	fallback_stylebox.unref();
 
+	if (default_theme_context) {
+		memdelete(default_theme_context);
+	}
+	while (theme_contexts.size()) {
+		HashMap<Node *, ThemeContext *>::Iterator E = theme_contexts.begin();
+		memdelete(E->value);
+		theme_contexts.remove(E);
+	}
+
 	singleton = nullptr;
 }
+
+void ThemeContext::_emit_changed() {
+	emit_signal("changed");
```

Comment:
```suggestion
	emit_signal(SNAME("changed"));
```
This is potentially emitted quite often.